core-rust/state-manager/src/protocol/genesis_data_resolver.rs

```diff
@@ -62,54 +62,112 @@
  * permissions under this License.
```

Comment:
lol, I find it funny that git actually categorized this as *.java -> *.rs rename + file edit :smile: